SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1384,7 +1396,8 @@ public async Task<IActionResult> YourRecommendations_Get(string reference)
             NumberOfPropertyRecommendations = propertyData.PropertyRecommendations.Count,
             HasEmailAddress = false,
             BackLink = GetBackUrl(QuestionFlowStep.YourRecommendations, propertyData),
-            EnergyPriceCapInfo = propertyData.GetEnergyPriceCapInfo()
+            EnergyPriceCapInfo = propertyData.GetEnergyPriceCapInfo(),
+            RecommendationsUpdatedSinceLastVisit = propertyData.RecommendationsUpdatedSinceLastVisit
```

Comment:
Similar question here

Reply:
this one should be fine since we update the properyData object and write it back out to the db when you put the code in, so we can retrieve it here

as in, `UpdatePropertyDataWithRecommendations` ends with
```cs
        await propertyDataStore.SavePropertyDataAsync(propertyData);
        return propertyData;
```